src/objects.rs

```diff
@@ -50,6 +50,12 @@ impl FromIterator<(String, String)> for Tags {
     }
 }
 
+impl<T: Iterator<Item = (String, String)>> From<T> for Tags {
+    fn from(f: T) -> Self {
+        FromIterator::from_iter(f)
+    }
+}
```

Comment:
That's not implemented for `Vec`, why? You already have '.collect()` that do the same.